### PR TITLE
EthereumProvider ES6

### PR DESCRIPTION
**Pull request by nivida**

### CODE REVIEW

File: packages/web3-core-method/src/models/methods/GetStorageAtMethodModel.js [hunk 1]
```diff
@@ -22,32 +22,30 @@
...
+export default class GetStorageAtMethodModel extends AbstractMethodModel {
+
+    /**
+     * @param {Object} utils
+     * @param {Object} formatters
+     *
+     * @constructor
+     */
+    constructor(utils, formatters) {
+        super('eth_getStorageAt', 3, utils, formatters);
```
Comment (frozeman) [inline packages/web3-core-method/src/models/methods/GetStorageAtMethodModel.js]: maybe a representation like:
```
 super(
    'eth_getStorageAt',
    3,
    utils,
    formatters
);
```
would improve readability of the models.

**Comment by frozeman** [comment]
You should add the babel conversion process as well.

**Comment by frozeman** [comment]
Your builds have a JSON error:
```
(/home/travis/build/ethereum/web3.js/node_modules/lerna/lib/PackageUtilities.js:131:22)
{ [JSONError: Unexpected token } while parsing '{    "name": "web3-eth",    "namespace' in packages/web3-eth/package.json]
  name: 'JSONError',
  fileName: 'packages/web3-eth/package.json',
  exitCode: 1 }
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.4
```

**Comment by nivida** [comment]
> You should add the babel conversion process as well.

I think babel will not be required because all "ethereum ready" platforms (browsers & node) are supporting es6. (https://kangax.github.io/compat-table/es6/)

**Pull request merged**